wandb/sdk/launch/docker.py

```diff
@@ -159,11 +162,15 @@ def build_docker_image(
         workdir_line=workdir_line,
         name_line=name_line,
     )
+
+    launch_metadata = create_launch_metadata(launch_project, docker_args, dockerfile)
```

Comment:
Is dockerfile a path or the actual contents?  It looks like we're setting the variable to a path on line 178 below...

Reply:
The actual contents, bad name, will fix that

Reply:
I'm also wondering, if I should refactor this to also include the actual docker command (with API key hidden of course)

Reply:
Yeah, sounds like a good idea.  I suppose a user could have other sensitive information in the command...  I suppose at some point in the future we could give users a way to mark certain parameters as sensitive.